theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/FocusRangeArea.tsx

```diff
@@ -47,44 +48,34 @@ const FocusRangeArea: React.FC<{
   return usePrism(() => {
     const existingRange = existingRangeD.getValue()
 
+    if (!existingRange) return null
+
     const range = existingRange?.range || {start: 0, end: 0}
 
     const height = val(layoutP.rightDims.height) + topStripHeight
 
-    let startPosInClippedSpace: number,
-      endPosInClippedSpace: number,
-      conditionalStyleProps:
-        | {
-            transform: string
-            background?: string
-          }
-        | undefined
-
-    if (existingRange !== undefined) {
-      startPosInClippedSpace = val(layoutP.clippedSpace.fromUnitSpace)(
-        range.start,
-      )
-
-      endPosInClippedSpace = val(layoutP.clippedSpace.fromUnitSpace)(range.end)
-
-      const desiredWidth = endPosInClippedSpace - startPosInClippedSpace
+    const startPosInClippedSpace = val(layoutP.clippedSpace.fromUnitSpace)(
+      range.start,
+    )
 
-      conditionalStyleProps = {
```

Comment:
These conditional styles are hard to grok. Better use conditional class names or just [styled component props](https://styled-components.com/docs/basics#passed-props) – more declarative and more idiomatic.